docs/integrations/builtin/core-nodes/n8n-nodes-base.extractfromfile.md

```diff
@@ -8,44 +8,72 @@ priority: high
 
 # Extract From File
 
-Use the Extract From File node to get data from a binary format and convert it to JSON.
+A common pattern in n8n workflows is to receive a file, either from and [HTTP Request node][] (for files you are fetching from a website), a [Webhook Node][] (for files which are sent to your workflow from elsewhere) or from a local source. Data obtained in this way is usually in a binary format, for example a spreadsheet or PDF.
+
+The Extract From File node extracts data from a binary format file and converts it to JSON, which can then be easily manipulated by the rest of your workflow.
 
-/// note | Convert to File
```

Comment:
This still feels like a useful note to me, but your call.

Reply:
I just didn't like it as an attention-grabbing  note right at the top, but yes, I did mean to put the info back somewhere